scripts/release/release-utils.js

```diff
@@ -61,7 +61,8 @@ function wrapInLi (content) {
  */
 function updateProjectPbxproj (projectPbxprojContent, commit) {
     const bskPackageRegex = new RegExp(
-        /(repositoryURL = "https:\/\/github\.com\/duckduckgo\/BrowserServicesKit";\s+requirement = {\s+kind = )(exactVersion)(;\s+)(version = \d+.\d+.\d+;)/
+        /(repositoryURL = "https:\/\/github\.com\/duckduckgo\/BrowserServicesKit";\s+requirement = {\s+kind = )(exactVersion)(;\s+)(version = \d+.\d+.\d+;)/,
+        'i'
```

Comment:
I've squeezed in these tiny changes to fix the release script. Someone changed the references in the iOS repo to use the capitalised `DuckDuckGo` and it broke our regex 🤦‍♂️. Now the regex is case insensitive and it should work again. I've changed the test to account for this in the future.